module/rdpClientCon.c

```diff
@@ -1145,7 +1145,13 @@ rdpClientConInit(rdpPtr dev)
             LLOGLN(0, ("rdpClientConInit: g_tcp_local_bind failed"));
             return 1;
         }
-        g_sck_listen(dev->listen_sck);
+        if (g_sck_listen(dev->listen_sck) != 0)
```

Comment:
This looks like an unrelated change. Please submit unrelated changes as separate commits, even if they are in the same pull request.